docs/source/developer_guide/plugins/update_plugin.rst

```diff
@@ -1,7 +1,7 @@
-Updating an Existing Plugin
-===========================
+Updating an Existing Plugin (or creating a new one)
+===================================================
 
-This section will take you through the steps needed to create a python package containing your plugin, which can then be easily developped, version controlled distributed and updated separately from aiida.
+This section will walk you through the steps needed to create a python package containing your plugin, which can then be easily developed, version controlled distributed and updated separately from AiiDA.
```

Comment:
revise the noun word "version controlled distributed". Maybe you want to say: "distributed through version controlled"